pom.xml

```diff
@@ -4,13 +4,12 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.cqfn.diktat</groupId>
     <artifactId>diktat-parent</artifactId>
-    <version>${diktat.version}</version>
+    <version>0.0.1-SNAPSHOT</version>
```

Comment:
it will be updated automatically?

Reply:
It will be updated by `mvn versions:set` in workflow, all modules at once.